src/Service.Host/client/src/actions/fetchMenuActions.js

```diff
@@ -4,7 +4,7 @@ import * as actionTypes from './index';
 
 const fetchMenu = () => ({
     [RSAA]: {
-        endpoint: `${config.proxyRoot}/intranet/menu-no-auth`,
+        endpoint: `https://app.linn.co.uk/intranet/menu-no-auth`,
```

Comment:
could use config.appRoot if you wanted

Reply:
is appRoot always the production url (app.linn.co.uk) regardless of environment?? This is the only place the menu is currently served up from with no auth issues..